src/licensedcode/models.py

```diff
@@ -768,6 +769,47 @@ def get_rules(
     licenses_as_rules = build_rules_from_licenses(licenses_db)
     return chain(licenses_as_rules, rules)
 
+def get_license_dirs(
+    additional_dirs,
+):
+    return [f"{path}/licenses" for path in additional_dirs]
+
+def get_rule_dirs(
+    additional_dirs,
+):
+    return [f"{path}/rules" for path in additional_dirs]
+
+def load_licenses_from_multiple_dirs(
+    license_directories,
+    with_deprecated=False,
+):
+    combined_licenses = {}
+    for license_dir in license_directories:
+        licenses = load_licenses(licenses_data_dir=license_dir, with_deprecated=False)
+        # this syntax for merging is described here: https://stackoverflow.com/a/26853961
+        combined_licenses = {**combined_licenses, **licenses}
+    return combined_licenses
+
+def get_rules_from_multiple_dirs(
+    licenses_db,
+    rule_directories,
+):
+    # TODO: error handling in case additional_directories is empty
+    # TODO: comment
+    if rule_directories:
+        rules_list = []
+        for rules_dir in rule_directories:
+            rules_list.append(list(load_rules(
```

Comment:
I think it's better to separate some of this out to their own variables for readability. 
```suggestion
            r = list(load_rules(rules_data_dir=rules_dir)
            combined_rules.append(r)
```